docs/apis/topaz_cli.md

```diff
@@ -17,6 +17,8 @@ These are
 `CodeCoverageCLI.bat` | Allows downloading results from a Xpediter Code Coverage repository by specifying the repository name, system name and test ID.
 `IspwCLI.bat` | Used for the Git to ISPW integration
 `SCMDownloaderCLI.bat` | Allows interaction with the ISPW downloader to download sources from ISPW repositories, Endevor repositories or PDS 
+`SCMDownloaderCLI.bat` | Allows interaction with the ISPW downloader to download sources from ISPW repositories, Endevor repositories or PDS
```

Comment:
Ratish, this looks like this line were duplicated. There seem to be other changes, I think they are all whitespace changes but this is not really our help. Would be safer to make sure that we are just adding our part